service/project.go

```diff
@@ -370,6 +370,48 @@ func (s *ProjectService) RejectInvitation(ctx context.Context, tkn cryptox.Token
 	return s.repo.DeleteInvitation(ctx, invitation.ID)
 }
 
+func (s *ProjectService) ListMembers(ctx context.Context, projectID, authUserID uuid.UUID) ([]model.ProjectMember, error) {
+	if err := s.authGuard.AuthorizeAdminRole(ctx, authUserID); err != nil {
+		return nil, err
+	}
+
+	exists, err := s.projectExists(ctx, projectID)
+	if err != nil {
+		return nil, err
+	}
+	if !exists {
+		return nil, apierrors.NewProjectNotFoundError()
+	}
+
+	return s.repo.ReadMembersForProject(ctx, projectID)
```

Comment:
When it comes to listing, perhaps a more correct implementation here is to query the members, and if it is empty check whether it is because the project does not exist (`404`) or simply there are indeed no members in the project (`200`).

Reply:
But it's a minor thing; and if it is in more places, I'd maybe leave it to a separate PR or maybe issue, because the opposite argument here is also that it's such a rare and non-critical case, that not worth it...